logictest/testdata/filter/filter

```diff
@@ -0,0 +1,26 @@
+createtable schema=default
+----
+
+insert cols=(labels.label1, labels.label2, labels.label3, labels.label4, stacktrace, timestamp, value)
+value1  value2  null    null    stack1  1   1
+value2  value2  value3  null    stack1  2   2
+value3  value2  null    value4  stack1  3   3
+----
+
+exec
+select labels.label1, labels.label2, labels.label3, labels.label4, stacktrace, timestamp, value where timestamp >= 2
```

Comment:
I think you might just be able to use `labels` here instead of `labels.label1, labels.label2, etc...`

Reply:
thanks, done